doc/contributing.rst

```diff
@@ -767,7 +801,7 @@ the release branch.
 4. Add release notes
```

Comment:
It would be good to update this section so it contains this information: https://docs.esmvaltool.org/en/latest/community/release_strategy.html#add-release-notes

Reply:
Good point. Now done in [bdd914f](https://github.com/ESMValGroup/ESMValCore/pull/1918/commits/bdd914fcdaa54214f94d12760a96c737c77e3d56)